pkg/controller/petset/pet_set.go

```diff
@@ -343,6 +343,11 @@ func (psc *PetSetController) syncPetSet(ps *apps.PetSet, pets []*api.Pod) (int,
 			if err == nil {
 				numPets++
 			}
+			switch err.(type) {
+			case *unhealthyPetBlock:
+				//We are not passing this error up, but we don't increment numPets if we encounter it
```

Comment:
please append: 
```
,since numPets directly translates to petset.status.replicas.
```
And leave a space after `//` and before `We` (I guess gofmt doesn't correct it, but that's consistent with everywhere else in the codebase).